wear/src/main/java/chat/rocket/android/main/ui/MainNavigator.kt

```diff
@@ -0,0 +1,46 @@
+package chat.rocket.android.main.ui
+
+import android.content.Intent
+import chat.rocket.android.R
+import chat.rocket.android.account.ui.AccountFragment
+import chat.rocket.android.chatroom.ui.chatRoomIntent
+import chat.rocket.android.chatrooms.ui.ChatRoomsFragment
+import chat.rocket.android.main.settings.ui.SettingsFragment.Companion.newInstance
+import chat.rocket.android.starter.ui.StarterActivity
+import chat.rocket.android.util.addFragment
+import chat.rocket.android.util.addFragmentBackStack
+
+class MainNavigator(internal val activity: MainActivity) {
```

Comment:
Add constants on the proper fragment classes with its tag instead of declaring `"ChatRoomsFragment"`, `"SettingsFragment"` and `"AccountFragment"`.